docs/quickstart/docker.md

```diff
@@ -30,39 +30,14 @@ Coder with Docker has the following advantages:
 1.  Run `coder login http://localhost:7080` in a new terminal and follow the
     interactive instructions to create your user.
 
-1.  Pull the example template:
+1.  Pull the example template using the interactive `coder templates init`:
```

Comment:
which example template do you want people to use here

Reply:
oops